yazi-fm/src/signals.rs

```diff
@@ -92,19 +92,19 @@ impl Signals {
 				if let Some(t) = &mut term {
 					select! {
 						biased;
-						Some(mut s) = rx.recv() => {
-							term = term.filter(|_| s.0);
-							s.1.take().map(|cb| cb.send(()));
+						Some((s, mut cb)) = rx.recv() => {
+							term = term.filter(|_| s);
+							cb.take().map(|cb| cb.send(()));
```

Comment:
I used `s` here because if we use `cb`, it will clash with the `cb` arg in `map(|cb|`, which lowers the recognizability